dotCMS/src/main/java/com/dotcms/rest/api/v1/temp/TempFileResource.java

```diff
@@ -67,7 +67,7 @@ public final Response uploadTempResourceMulti(@Context final HttpServletRequest
             if (!Config.getBooleanProperty(TempFileAPI.TEMP_RESOURCE_ENABLED, true)) {
                 final String message = "Temp Files Resource is not enabled, please change the TEMP_RESOURCE_ENABLED to true in your properties file";
                 Logger.error(this, message);
-                throw new DotSecurityException(message);
+                throw new DoesNotExistException(message);
```

Comment:
Very easy refactor: Let's extract the entire if in a private method call `verifyTempResourceEnabled()` and replace in the method above also

Reply:
done